.github/workflows/pr.yml

```diff
@@ -136,5 +140,6 @@ jobs:
       - nvrtc
       - thrust
       - examples
+      - verify-devcontainers
```

Comment:
I am not convinced that this should run on every PR

Reply:
Strictly speaking, it doesn't. However, the job is fast relative to any of the other jobs and it isn't really worth the effort to set up the logic to only run when certain files are changed.